onnxruntime/test/testdata/required_ops.config

```diff
@@ -5,6 +5,7 @@ ai.onnx;9;Abs,Add,BatchNormalization,Cast,Clip,Concat,Constant,ConstantOfShape,C
 ai.onnx;10;Add,Cast,Concat,ConstantOfShape,Div,Dropout,Erf,Expand,Gather,Greater,Identity,If,LayerNormalization,Loop,MatMul,Mul,Neg,NonZero,Pow,ReduceMean,ReduceSum,Shape,Sqrt,Squeeze,Sub,Tanh,Transpose,Unsqueeze
 ai.onnx;11;Abs,Add,ArgMax,BatchNormalization,Cast,Clip,Concat,Constant,ConstantOfShape,Conv,Div,Exp,Expand,Flatten,Gather,Gemm,Identity,If,LayerNormalization,Log,Loop,MatMul,MatMulInteger,Max,Min,Mul,Neg,Pow,RandomUniform,Range,ReduceMean,ReduceSum,ReduceSumSquare,Relu,Reshape,Scan,SequenceConstruct,SequenceInsert,SequenceLength,Shape,Sigmoid,Slice,Softmax,Split,Sqrt,Squeeze,Sub,Sum,Tanh,Transpose,Unsqueeze,Where
 ai.onnx;12;Add,And,Cast,Concat,Constant,ConstantOfShape,Conv,CumSum,Div,Dropout,DynamicQuantizeLinear,Equal,Erf,Expand,Flatten,Gather,GatherND,Gemm,GlobalAveragePool,Identity,If,LayerNormalization,Less,Loop,MatMul,MatMulInteger,Min,Mul,NonZero,Not,Pad,Pow,RandomNormalLike,Range,ReduceMean,ReduceSum,Relu,Reshape,Shape,Slice,Softmax,SoftmaxCrossEntropyLoss,SparseSoftmaxCrossEntropy,Split,Sqrt,Squeeze,Sub,Tanh,Transpose,Unsqueeze,Where
+ai.onnx;14;Trilu
 ai.onnx;314159;Add
 ai.onnx.ml;1;ArrayFeatureExtractor,DictVectorizer,LabelEncoder,LinearClassifier,Normalizer,OneHotEncoder,TreeEnsembleRegressor,ZipMap
-com.microsoft;1;BiasGelu,Fake_FunctionOp,FastGelu,FusedGemm,FusedMatMul,Gelu,Trilu
+com.microsoft;1;BiasGelu,Fake_FunctionOp,FastGelu,FusedGemm,FusedMatMul,Gelu
```

Comment:
Honestly I don't know which one (contri_ops or ONNX domain) is better for this usage.